web/user_profile/serializers.py

```diff
@@ -125,3 +127,12 @@ def save(self, **kwargs):
         user_data = dict(data)
         User.objects.filter(id=kwargs.get("id")).update(**user_data)
         models.Profile.objects.filter(user__pk=kwargs.get("id")).update(**profile_data)
+        EmailAddress.objects.filter(user__pk=kwargs.get("id")).update(email=user_data['email'])
```

Comment:
if you want to have an opportunity to change email, you need to have a flow with deactivation old email and confirmation new email

Reply:
made deactivation and confirmation flow